python2/httplib2test.py

```diff
@@ -263,7 +254,7 @@ def testGetConnectionRefused(self):
         self.assertEqual(response.status, 400)
 
     def testGetIRI(self):
-        if sys.version_info >= (2,3):
+        if sys.version_info >= (2, 7):
```

Comment:
For python2/httplib2test.py there is no other version except 2.7, so please remove this condition.